liquibase-extension-testing/src/main/resources/META-INF/services/liquibase.command.CommandStep

```diff
@@ -0,0 +1,2 @@
+liquibase.extension.testing.testsystem.command.TestSystemDownCommand
```

Comment:
If you are interested, we used a maven plugin in pro that generates these files dynamically at build time. I can point you to more info, let me know.

Reply:
Yeah, it's worth thinking about at some point but not as part of this PR